content/docs/user-guide/components/reference/ros2/_index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: ROS 2 Components
+linktitle: ROS 2
+description: Using Robot Operating System (ROS 2) components in Open 3D Engine (O3DE).
+---
+
+Refer to the [Robotics](/docs/user-guide/interactivity/robotics) documentation to learn more about robotics simulation features of O3DE.
+
+ROS 2 Gem components can be divided into the following categories:
+-  [Core components](core), essential to running simulations independent on type of robot. 
```

Comment:
...essential to running simulations independent **of the** type of robot.

Reply:
Thanks, will apply!